zebra-network/src/peer_set/initialize/tests/vectors.rs

```diff
@@ -1351,7 +1351,12 @@ async fn add_initial_peers_deadlock() {
 
     let nil_inbound_service = service_fn(|_| async { Ok(Response::Nil) });
 
-    let init_future = init(config, nil_inbound_service, NoChainTip);
+    let init_future = init(
+        config,
+        nil_inbound_service,
+        NoChainTip,
+        "Test user agent".to_string(),
```

Comment:
Same question as above: why not the actual user agent?

Reply:
Explained in https://github.com/ZcashFoundation/zebra/pull/6601#discussion_r1185440905.